catalog/transformer/catalog-transformer-streaming-impl/src/main/java/org/codice/ddf/transformer/xml/streaming/impl/Gml32ToWktImpl.java

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.transformer.xml.streaming.impl;
+
+import org.geotools.gml3.v3_2.GMLConfiguration;
+import org.geotools.xml.Parser;
+
+public class Gml32ToWktImpl extends Gml3ToWktImpl {
+  public Gml32ToWktImpl() {
+    ClassLoader tccl = Thread.currentThread().getContextClassLoader();
```

Comment:
Geotools uses Java SPI. When instantiated from blueprint SPI would get a different classloader than expected and failed to find its services. I had to break the implementation out into two separate classes since we couldn't use blueprint injection.